index.d.ts

```diff
@@ -101,7 +101,7 @@ declare module 'focus-trap' {
    *  find the element.
    */
   export function createFocusTrap(
-    element: HTMLElement | string,
+    element: HTMLElement | string | Array<HTMLElement | string>,
```

Comment:
Please also add the new `updateContainerElements()` method to the `FocusTrap` interface.

Reply:
Good call.  Done.